pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/users/models/Guid.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.ddp.json.users.models;
+
+import lombok.Value;
+
+@Value
+public class Guid {
+    private String value;
+
+    public String toString() {
+        return value;
+    }
+}
```

Comment:
This class mostly just serves as explicit indicator of the content type. There are multiple types of GUIDs that we use, and they don't have explicit indicators what each one is used for, so there isn't much we can do here in the way of validation.